test/src/auth/SignInWithPassword.tsx

```diff
@@ -20,6 +22,8 @@ export function SignInWithPassword({
   const { signIn } = useAuthActions();
   const [flow, setFlow] = useState<"signIn" | "signUp">("signIn");
   const { toast } = useToast();
+  const [passwordRequirementsColor, setPasswordRequirementsColor] =
+    useState("text-gray-500");
```

Comment:
We should not encourage this pattern (this code is linked from docs).

Instead use something like `const [error, setError]` and keep the mapping to colors in the JSX.

UX-wise though it might be better to show a different toast, because the color won't change on subsequent submits. This will avoid the need for more state altogether.

Reply:
Sounds good, switched to using the toast for simplicity.